samza-api/src/main/java/org/apache/samza/operators/MessageStream.java

```diff
@@ -49,25 +49,51 @@
 @InterfaceStability.Evolving
 public interface MessageStream<M> {
 
+  /**
+   * Equivalent to {@code map("", mapFn)}.
+   * @see #map(String, MapFunction)
+   */
+  default <OM> MessageStream<OM> map(MapFunction<? super M, ? extends OM> mapFn) {
+    return map("", mapFn);
+  }
+
   /**
    * Applies the provided 1:1 function to messages in this {@link MessageStream} and returns the
    * transformed {@link MessageStream}.
    *
+   * @param desc description of this mapping operation
```

Comment:
Since this is included in logs and metric names, let's call out that this should be a "concise name" for the operator, not a "description". Let's also mention why users should provide this ("... to be included in metrics and logs.")

Reply:
+1 "tag" or "name" might be better.

Reply:
Renamed to "name"